### PR TITLE
Explain how to create Auth0 APIs on native QS

### PR DESCRIPTION
**Pull request by lbalmaceda**
I added some common texts on the `_includes` folder so you can reuse if needed from the swift QS @cocojoe.

### CODE REVIEW

File: articles/quickstart/native/android/06-calling-apis.md [hunk 1]
```diff
@@ -22,23 +25,53 @@ This tutorial shows you how to use a previously saved token to authenticate your
 
 Before you continue with this tutorial, make sure that you have completed the previous tutorials. This tutorial assumes that:
 * You have completed the [Session Handling](/quickstart/native/android/03-session-handling) tutorial and you know how to handle the `Credentials` object.
-* You have set up a backend application as API. To learn how to do it, follow one of the [backend tutorials](https://auth0.com/docs/quickstart/backend). 
+* You have set up a backend application as API. To learn how to do it, follow one of the [backend tutorials](https://auth0.com/docs/quickstart/backend).
```
Comment (jeffreylees) [inline articles/quickstart/native/android/06-calling-apis.md]: `[backend tutorials](/quickstart/backend)`

File: articles/quickstart/native/android/06-calling-apis.md [hunk 2]
```diff
@@ -22,23 +25,53 @@ This tutorial shows you how to use a previously saved token to authenticate your
...
+
 
-After you set up an API, declare the endpoint you need to call as a constant in the current class:
+<%= include('_includes/_calling_api_create_api') %>
+
+<%= include('_includes/_calling_api_create_scope') %>__
```
Comment (jeffreylees) [inline articles/quickstart/native/android/06-calling-apis.md]: stray underscores after your include?

**Review by jeffreylees — CHANGES_REQUESTED** [review]
Not much to say here, looks good to me other than these couple of notes.

File: articles/quickstart/native/android/06-calling-apis.md [hunk 3]
```diff
@@ -86,6 +119,6 @@ client.newCall(request).enqueue(new Callback() {
 });
 ```
 
-Check if the request was made and if the response that came back was what you expected. 
+Check if the request was made and if the response that came back was what you expected.
 
-You need to configure your server side to protect your API endpoints with the key for your Auth0 client. In this example, you can use the user's access token issued by Auth0 to call Auth0 APIs.
\ No newline at end of file
+You need to configure your backend-app to protect your API endpoints with the key for your Auth0 client, API identifier and API scopes. In this example, you can use the user's access token issued by Auth0 to call your own APIs.
```
Comment (cocojoe) [inline articles/quickstart/native/android/06-calling-apis.md]: Pick either `backend-app` or `backend application` and use consistently throughout.  I prefer `backend application`

File: articles/quickstart/native/android/06-calling-apis.md [hunk 4]
```diff
@@ -86,6 +119,6 @@ client.newCall(request).enqueue(new Callback() {
 });
 ```
 
-Check if the request was made and if the response that came back was what you expected. 
+Check if the request was made and if the response that came back was what you expected.
```
Comment (cocojoe) [inline articles/quickstart/native/android/06-calling-apis.md]: Is this information needed? the code is commented.

File: articles/quickstart/native/android/06-calling-apis.md [hunk 5]
```diff
@@ -22,23 +25,53 @@ This tutorial shows you how to use a previously saved token to authenticate your
...
 
-After you set up an API, declare the endpoint you need to call as a constant in the current class:
+<%= include('_includes/_calling_api_create_api') %>
+
+<%= include('_includes/_calling_api_create_scope') %>__
+
+
+## Get the User's Access Token
+
+Get the user's access token ready for your API to consume by requesting as audience the API identifier used when creating the Auth0 API. Do so by using the `withAudience` method of the WebAuthProvider builder. For instructions, see the [Login](/quickstart/native/android/00-login) tutorial. Add at the top of the class the constants for the backend-app API secure URL and the API identifier.
```
Comment (cocojoe) [inline articles/quickstart/native/android/06-calling-apis.md]: `Get the user's access token ready for your API to consume by requesting as audience the API identifier used when creating the Auth0 API. Do so by using the `withAudience` method of the WebAuthProvider builder.` ->
`To retrieve an access token that is authorized to access your API, you need to specify the API Identifier you created in the Auth0 dashboard.`

`Add at the top of the class the constants for the backend-app API secure URL and the API identifier.` ->
`At the top of the class add the constants for accessing the API, **API_URL** and **API_IDENTIFIER**

**Review by cocojoe — CHANGES_REQUESTED** [review]
Once this is merged, I'll refactor into iOS. Thx

File: articles/quickstart/native/android/06-calling-apis.md (hunk 2 above)
Comment (lbalmaceda) [reply articles/quickstart/native/android/06-calling-apis.md]: With underscores
image

Without underscores
image

I know then don't belong there, but they don't hurt also and help syntax parsers like the Atom's.

Comment (jeffreylees) [reply articles/quickstart/native/android/06-calling-apis.md]: Haha, I might have to start doing that myself then. As long as it won't bother anything when it builds it doesn't bother me.

File: articles/quickstart/native/android/06-calling-apis.md (hunk 4 above)
Comment (lbalmaceda) [reply articles/quickstart/native/android/06-calling-apis.md]: not really. n1

File: articles/quickstart/native/android/06-calling-apis.md (hunk 5 above)
Comment (lbalmaceda) [reply articles/quickstart/native/android/06-calling-apis.md]: I changed this a bit

File: articles/quickstart/native/android/06-calling-apis.md (hunk 2 above)
Comment (lbalmaceda) [reply articles/quickstart/native/android/06-calling-apis.md]: It shouldn't as it's only "opening italics" and "closing italics". You can add a space in the middle, or put it in the line below. All are fine 😝

Comment (lbalmaceda) [reply articles/quickstart/native/android/06-calling-apis.md]: This just happens because we use a non-md syntax with the includes.

**Review by cocojoe — APPROVED** [review]

**Comment by cocojoe** [comment]
@jeffreylees anything blocking you here?

**Review by jeffreylees — APPROVED** [review]

**Comment by jeffreylees** [comment]
Nope sorry all good

File: articles/quickstart/native/android/06-calling-apis.md (hunk 2 above)
Comment (cocojoe) [reply articles/quickstart/native/android/06-calling-apis.md]: @lbalmaceda These links are wrong, this links to to the _include folder in the Android directory. should be `../_includes`

**Comment by lbalmaceda** [comment]
fixed include paths.

**Review by cocojoe — APPROVED** [review]
I don't like the trailing underscores, one day perhaps the include works differently and replaces only the content in the tags vs the entire line.

**Comment by lbalmaceda** [comment]
Underscores could be removed but the code highlight will stop working.. The include doesn't replace the line but rather the tag. I could put the underscores 2 lines below and it will continue to work fine. Because there are 2 underscores, the MD syntax is "opening and closing the italics". TL;DR: The rest of the article remains unaffected. 
This is because of the naming convention `docs` took regarding include files.

**Review by aaguiarz — APPROVED** [review]

**Pull request merged**